test/extended/openstack/loadbalancers.go

```diff
@@ -141,14 +145,108 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb] The Openstack
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(isLbMethodApplied(lbMethod, results, pods)).Should(o.BeTrue(), "%q lb-method not applied after 100 queries:\n%v\n", lbMethod, results)
 	})
+
+	for _, i := range availableLbProvidersUnderTests {
+		lbProviderUnderTest := i
+		// https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md#sharing-load-balancer-with-multiple-services
+		g.It(fmt.Sprintf("should re-use an existing UDP %s LoadBalancer when new svc is created on Openshift with the proper annotation", lbProviderUnderTest), func() {
+
+			g.By("Checking cluster configuration")
+			cloudProviderConfig, err := getConfig(oc.AdminKubeClient(),
+				"openshift-config",
+				"cloud-provider-config",
+				"config")
+			o.Expect(err).NotTo(o.HaveOccurred())
+			lbProvider, err := getClusterLoadBalancerSetting("lb-provider", cloudProviderConfig)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			if lbProvider != strings.ToLower(lbProviderUnderTest) {
+				e2eskipper.Skipf("Test not applicable for LoadBalancer provider different than %s: %q", lbProviderUnderTest, lbProvider)
+			}
+			setting, err := getClusterLoadBalancerSetting("max-shared-lb", cloudProviderConfig)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			maxSharedLb, err := strconv.Atoi(setting)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			if 2 < maxSharedLb {
+				e2eskipper.Skipf("Test not applicable when max-shared-lb is lower than 2 and it is %s", maxSharedLb)
```

Comment:
looks like maxSharedLb is an int and can't be included in the string

Reply:
Changed to %d. Done